files/scripts/bgp.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+SERVICE="bgp"
+DEBUGLOG="/tmp/bgp_debug.log"
+LOCKFILE="/tmp/bgp-lock"
+
+function debug()
+{
+		/usr/bin/logger $1
+		/bin/echo `date` "- $1" >> ${DEBUGLOG}
+}
+
+function lock_service_state_change()
+{
```

Comment:
lock and unlock is not needed for BGP. syncd/swss needed it because these 2 services needs to start/stop together.

Reply:
removed as suggested